### DISCUSSION TITLE
Branding capitalization

### DISCUSSION THREAD

**Discussion by hello-smile6** [JaydenDev/freecat#38] [Q&A]
Should `Freecat` in the window name use the existing capitalization (`Freecat`), or match `home.html` (`FreeCat`)? Just wondering.

**Comment by hello-smile6** [comment]
@JaydenDev @webdev03 Any input?

**Reply by imdevarsh** [reply]
I'd say to use `Freecat` but I'm not sure

**Reply by hello-smile6** [reply]
Okay, let's migrate this to your fork for now.

**Reply by hello-smile6** [reply]
> I'd say to use `Freecat` but I'm not sure

Could you enable GitHub Discussions?
